tests/sample_module.py

```diff
@@ -19,7 +19,7 @@ class SomeClass:
     other_class_attribute = OtherClass
     typedattr: str = "bruh"
 
-    def method(self):
+    def method(self, *args, **kwargs):
```

Comment:
This was used for writing a different test case which I removed before committing. The removed test was to test the "correct usage" of the `AnyInstanceOf `using `mock_callable`. If asked to, I'll add that test again.